esmvalcore/cmor/_fixes/ipslcm/ipsl_cm6.py

```diff
@@ -0,0 +1,103 @@
+"""Fixes for IPSLCM6 TS output format."""
+import logging
+import subprocess
+import time
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+logger = logging.getLogger(__name__)
+
+# The key used in extra_facets file for providing the
+# variable name (in NetCDF file) that match the CMOR variable name
+VARNAME_KEY = "ipsl_varname"
+
+
+class AllVars(Fix):
+    """Fixes for all IPSLCM variables."""
+
+    def fix_file(self, filepath, output_dir):
+        """Select IPSLCM variable in filepath, by calling CDO, if relevant.
```

Comment:
```suggestion
        """Select IPSLCM variable in filepath.
```
The CDO implementation detail is not needed in the short summary. I would probably leave it out of the documentation completely (why does the user need to know?), but if you want to mention it a note at the end of the docstring seems better.

Reply:
The user may need to know that, in a future version, using CDO may become less useful.
I moved these few words out of the short summary